src/net_processing.cpp

```diff
@@ -124,7 +124,9 @@ static const unsigned int NODE_NETWORK_LIMITED_MIN_BLOCKS = 288;
 /** Average delay between local address broadcasts */
 static constexpr auto AVG_LOCAL_ADDRESS_BROADCAST_INTERVAL = 24h;
 /** Average delay between peer address broadcasts */
-static constexpr auto AVG_ADDRESS_BROADCAST_INTERVAL = 30s;
+static constexpr std::chrono::seconds AVG_ADDRESS_BROADCAST_INTERVAL{30};
```

Comment:
why is this changed? This also conflicts with #23832

Reply:
I think it was like that from the very beginning? But sure, I can drop this.

Reply:
Using the `static constexpr auto NAME = <numer><suffix>;` as is used here is a bit more readable I think. I'd suggest using the same for the line below.